cpp/log/cert.cc

```diff
@@ -243,24 +243,15 @@ unique_ptr<Cert> Cert::FromDerBio(BIO* bio_in) {
 
 
 string Cert::PrintVersion() const {
-  if (!IsLoaded()) {
-    LOG(ERROR) << "Cert not loaded";
-    return string();
-  }
-
-  const long version(X509_get_version(x509_.get()));
+  const long version(X509_get_version(CHECK_NOTNULL(x509_.get())));
```

Comment:
Well, to be fair, if it doesn't `CHECK`-fail here, it'll just segfault with a slightly less clear error message. :wink: